vcpkg.json

```diff
@@ -46,20 +47,23 @@
 			"default-features": false
 		},
 		{
-			"name": "lilv",
-			"default-features": false
+			"name": "libxft",
+			"default-features": false,
+			"platform": "linux"
 		},
 		{
-			"name": "lv2",
-			"default-features": false
+			"name": "libxinerama",
+			"default-features": false,
+			"platform": "linux"
 		},
 		{
 			"name": "mp3lame",
 			"default-features": false
 		},
 		{
-			"name": "portaudio",
```

Comment:
Removing this line might disable portaudio.

Reply:
I'll fix this too